duplicate_files.py

```diff
@@ -28,7 +28,7 @@ def main(w):
 
```

Comment:
Please verify the following diffs:
- [ ] Undocumented templates: [en](https://wiki.tf/d/3104537)
- [ ] All articles: [en](https://wiki.tf/d/3104537) [ar](https://wiki.tf/d/3104537) [cs](https://wiki.tf/d/3104537)